test/fixtures/get-pathfind-response.json

```diff
@@ -0,0 +1,65 @@
+    "allowPartialPayment": false,
+    "noDirectRipple": false
+  },
+  {
+    "source": {
+      "address": "r9cZA1mLK5R5Am25ArfXFmqgNwjZgnfk59",
+      "amount": {
+        "currency": "USD",
+        "value": "0.001002",
+        "counterparty": "r9cZA1mLK5R5Am25ArfXFmqgNwjZgnfk59"
+      }
+    },
+    "destination": {
+      "address": "rpZc4mVfWUif9CRoHRKKcmhu1nx2xktxBo",
+      "amount": {
+        "currency": "USD",
+        "counterparty": "rMH4UxPrbuMa1spCBR98hLLyNJp4d8p4tM",
+        "value": "100"
+      }
+    },
+    "paths": "[[{\"account\":\"rMH4UxPrbuMa1spCBR98hLLyNJp4d8p4tM\",\"type\":1,\"type_hex\":\"0000000000000001\"}],[{\"account\":\"rMwjYedjc7qqtKYVLiAccJSmCwih4LnE2q\",\"type\":1,\"type_hex\":\"0000000000000001\"},{\"currency\":\"USD\",\"issuer\":\"rMH4UxPrbuMa1spCBR98hLLyNJp4d8p4tM\",\"type\":48,\"type_hex\":\"0000000000000030\"},{\"account\":\"rMH4UxPrbuMa1spCBR98hLLyNJp4d8p4tM\",\"type\":1,\"type_hex\":\"0000000000000001\"}],[{\"account\":\"rMwjYedjc7qqtKYVLiAccJSmCwih4LnE2q\",\"type\":1,\"type_hex\":\"0000000000000001\"},{\"currency\":\"XRP\",\"type\":16,\"type_hex\":\"0000000000000010\"},{\"currency\":\"USD\",\"issuer\":\"rMH4UxPrbuMa1spCBR98hLLyNJp4d8p4tM\",\"type\":48,\"type_hex\":\"0000000000000030\"},{\"account\":\"rMH4UxPrbuMa1spCBR98hLLyNJp4d8p4tM\",\"type\":1,\"type_hex\":\"0000000000000001\"}],[{\"account\":\"rMwjYedjc7qqtKYVLiAccJSmCwih4LnE2q\",\"type\":1,\"type_hex\":\"0000000000000001\"},{\"currency\":\"XRP\",\"type\":16,\"type_hex\":\"0000000000000010\"},{\"currency\":\"USD\",\"issuer\":\"rpHgehzdpfWRXKvSv6duKvVuo1aZVimdaT\",\"type\":48,\"type_hex\":\"0000000000000030\"},{\"account\":\"rpHgehzdpfWRXKvSv6duKvVuo1aZVimdaT\",\"type\":1,\"type_hex\":\"0000000000000001\"},{\"account\":\"rMH4UxPrbuMa1spCBR98hLLyNJp4d8p4tM\",\"type\":1,\"type_hex\":\"0000000000000001\"}]]",
```

Comment:
Do you know what they are for? Not in the documentation: https://github.com/ripple/ripple-dev-portal/issues/126

I would assume they are necessary, but if not, that would be something to address in rippled.